### DISCUSSION TITLE
update the stable version of the A320

### DISCUSSION THREAD

**Discussion by yuriks59** [flybywiresim/aircraft#3524] [General]
Friends, it would be good to update the stable version of the A320

**Comment by riveroalbert** [comment]
Patience! Milestone to v0.6.0 is 70% complete! https://github.com/flybywiresim/a32nx/milestone/7

**Reply by tomber42** [reply]
How many stable versions are planned within a year?

**Comment by tomber42** [comment]
maybe it would be good to freeze 0.60 now so that testing could be finalized. At the moment every day new PR comes to 0.60

**Comment by hotshotp** [comment]
We will update it when its ready

**Reply by wernair2k21** [reply]
Take your time and thanks for the great work - really appreciate your effort!
